pre-docs/inventory_file.md

```diff
@@ -58,7 +58,7 @@ groups:
 
 ```
 
-## Override a user for a specific node
+### Override a user for a specific node
```

Comment:
This is another one of those heading level issues that I can't easily change. Is it okay to leave this at the same level for now? I think at some point I'll have to fix this limitation in docs...

Reply:
Ok, leave it how it was. It just seemed more like an example of using inventory rather than a completely new section.